src/components/OrderDetail/OrderCustomerInfo.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {format} from '@/utils/date';
+import S from './OrderCustomerInfo.style';
+
+type Props = {
+  id: string;
+  orderMemberName: string;
+  createdAt: string;
+  pickupReservedAt: string;
+  customerRequest: string;
+};
+
+const OrderCustomerInfo = ({
+  id,
+  orderMemberName,
+  createdAt,
+  pickupReservedAt,
+  customerRequest,
+}: Props) => {
+  return (
+    <S.Container>
+      <S.InfoText>주문번호: {id}</S.InfoText>
+      <S.InfoText>주문자명: {orderMemberName}</S.InfoText>
+      <S.InfoText>
+        주문 일시:{' '}
+        {format(new Date(createdAt).getTime(), 'YYYY. MM. DD. (ddd) A hh:mm')}
+      </S.InfoText>
+      <S.InfoText>
+        픽업 예정 시간:{' '}
+        {format(
+          new Date(pickupReservedAt).getTime(),
+          'YYYY. MM. DD. (ddd) A hh:mm',
+        )}
```

Comment:
[문자열 템플릿 이터럴](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Template_literals) 을 활용해서 jsx 안의 문자열을 표현함이 어떨까요. 
`주문 일시:{' '}` 에서 화이트 스페이스가 prettier에 의해서 생기는 현상이랑 가독성에 도움이 될 것 같습니다.

```suggestion
      <S.InfoText>{`주문번호: ${id}`}</S.InfoText>
      <S.InfoText>{`주문자명: ${orderMemberName}`}</S.InfoText>
      <S.InfoText>
        {`주문 일시: ${format(new Date(createdAt).getTime(), 'YYYY. MM. DD. (ddd) A hh:mm')}`}
      </S.InfoText>
      <S.InfoText>
        {`픽업 예정 시간: ${format(new Date(pickupReservedAt).getTime(), 'YYYY. MM. DD. (ddd) A hh:mm')}`}
```